src/github.com/getlantern/enproxy/conn_impl.go

```diff
@@ -17,7 +18,7 @@ import (
 // addr: the host:port of the destination server that we're trying to reach
 //
 // config: configuration for this Conn
-func Dial(addr string, config *Config) (*Conn, error) {
+func Dial(addr string, config *Config) (net.Conn, error) {
```

Comment:
Dial now returns a net.Conn rather than our concrete Conn, since the client doesn't need it.